docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following known issues.
+
+.Lack of unique IDs and Aria-labels on sections and categories hinders automation
+
+There is a lack of unique IDs applied to sections and categories, which hinders migration automation. link:https://issues.redhat.com/browse/MTA-1203[MTA-1203]
+
+.Support disconnected installations
+
+MTA does not support disconnected installations. link:https://issues.redhat.com/browse/MTA-17[MTA-17]
+
+.Cannot remove the Jira connection to a migration wave that is later removed 
+
+It is not possible to remove a Jira connection to a migration wave after that wave is removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
+
+.Binary analysis takes three hours and then fails
+
+Binary analysis can take three hours, and then it fails. link:https://issues.redhat.com/browse/MTA-1183[MTA-1183]
+
+.Allow architects to review an application without an assessment
+
+The MTA user can decide to skip the Assessment stage of the application migration. link:https://issues.redhat.com/browse/MTA-115[MTA-115]
```

Comment:
I am sorry but this is badly written

Address directly

Reply:
changed